code/controllers/Processes/playtime.dm

```diff
@@ -0,0 +1,10 @@
+/datum/controller/process/playtime/setup()
```

Comment:
Someone could change jobs or go AFK during the middle of the round, and having a process ensures that the time recorded is a relatively accurate measurement of someone's time in a department.